.github/workflows/ci.yml

```diff
@@ -0,0 +1,48 @@
+name: Go
+
+on:
+  push:
+    branches:
```

Comment:
As you pointed out in chat, the CI is not running for this PR. Maybe the triggers are not working for pull requests from forks? You can check the docu here: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#on .
If you don't specify any branches, you can be more concise like
```on: [push, pull_request]```